applications/nrf_desktop/src/events/config_event.h

```diff
@@ -60,7 +60,6 @@ extern "C" {
 						       MOD_FIELD_SET(module) | OPT_FIELD_SET(option))
 
 #define SETUP_MODULE_SENSOR	0x1
-#define SETUP_MODULE_LED	0x2
```

Comment:
This define is removed, is it not used?

Reply:
It was used before by `led_stream`, then the implementation changed and it was not removed.

Reply:
ok, then - I was afraid that something was removed by accident